app/src/main/java/uk/gov/android/authentication/login/LoginSession.kt

```diff
@@ -30,6 +30,7 @@ interface LoginSession {
     @Throws(Exception::class)
     fun finalise(
         intent: Intent,
+        appIntegrity: Pair<String?, String?>,
```

Comment:
Maybe some documentation on the `LoginSession` generally whilst we're here

Reply:
I added in KDoc explanation in the LoginSession interface of how this new param is being used and I changed it to a data class which looks fairly clear. After you finish reviewing I'll push my changes so you can have a look